### DISCUSSION TITLE
How to add more video source to the application

### DISCUSSION THREAD

**Discussion by xianghui-renesas** [webcamoid/webcamoid#589] [General]
The add source option takes an URL as input. Does this mean you can provide a YouTube URL to display on the webcamoid application as an application source? I tried it, it does not seem to work. Any comment is appreciated!

**Comment by hipersayanX** [comment]
No, URL source only takes raw video sources, it doesn't works with Youtube, Twitch, Kick or whatever. But it could be a feature in a very distant future.
